events_emitter/example-settings.py

```diff
@@ -41,10 +43,14 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-    'events_emitter'
+    'events_emitter',
+    'django_celery_results',
+    'django_celery_beat',
+    'eventful_django',
```

Comment:
Please, add Prometheus to the service settings and service dependencies in production.